SofaKernel/framework/sofa/simulation/AnimateVisitor.cpp

```diff
@@ -190,6 +190,9 @@ Visitor::Result AnimateVisitor::processNodeTopDown(simulation::Node* node)
             end(node, node->solver[i], t0);
         }
 
+        MechanicalProjectPositionAndVelocityVisitor(&m_mparams, nextTime,
```

Comment:
It looks like a wrong code duplication.

please keep the `#ifdef SOFA_SUPPORT_MAPPED_MASS` version.